scripts/ingests/2020Best_publications_ingest.py

```diff
@@ -0,0 +1,71 @@
+from operator import add
+from re import search
+import sys
+from sqlalchemy.sql.elements import Null
+sys.path.append('.')
+from scripts.ingests.utils import *
+from astrodbkit2.astrodb import create_database
+from astrodbkit2.astrodb import Database
+from simple.schema import *
+from astropy.table import Table
+from pathlib import Path
+import os
+
+save_db = False #modifies .db file but not the data files
+RECREATE_DB = False #recreates the .db file from the data files
+VERBOSE = False
+
+verboseprint = print if VERBOSE else lambda *a, **k: None
+
+db_file = 'SIMPLE.db'
+db_file_path = Path(db_file)
+db_connection_string = 'sqlite:///SIMPLE.db'  # SQLite browser
+
+if RECREATE_DB and db_file_path.exists():
+    os.remove(db_file) #removes the current .db file if one already exists
+
+if not db_file_path.exists():
+    create_database(db_connection_string) #creates empty database based on the simple schema
+    db = Database(db_connection_string) #connects to the empty database
+    db.load_database('data/') #loads the data from the data files into the database
+else:
+    db = Database(db_connection_string) #if database already exists, connects to .db file
+
+
+# add missing references
+#Searching for all publications in table and adding missing ones to pub table in .db file
+Pubs = Table.read('scripts/ingests/UltracoolSheet-References.csv', data_start=1, data_end=20)
+bibcodes = Pubs['ADSkey_ref']
+for i,ref in enumerate(Pubs['code_ref']):
+	if search_publication(db, name=ref) == False:
```

Comment:
I think we should get rid of this first `if`.  Just keep the next one. In other words, if the bibcode doesn't exist in the database, add it.